autogpt_platform/backend/.env.example

```diff
@@ -20,13 +20,13 @@ PYRO_HOST=localhost
 SENTRY_DSN=
 
 ## User auth with Supabase is required for any of the 3rd party integrations with auth to work.
-ENABLE_AUTH=false
+ENABLE_AUTH=true
```

Comment:
I ageee with this change. I think we should take it one step further and just make it the only option

Reply:
I guess the local test might still need this, I don't think the integration test will need these humongous just for auth

Reply:
We do it for a few other tests on frontend unfortunately. May as well commit to the bit since all of them will have to pass anyway  not going to block on this point but worth considering as it’s a goal of the Automated Testing project

Reply:
I think there is already a ticket for this somewhere. But yes I agree.